node_package/src/serverRenderReactComponent.ts

```diff
@@ -33,37 +32,42 @@ See https://github.com/shakacode/react_on_rails#renderer-functions`);
     });
 
     const processServerRenderHash = () => {
-        // We let the client side handle any redirect
-        // Set hasErrors in case we want to throw a Rails exception
-        hasErrors = !!(reactRenderingResult as {routeError: Error}).routeError;
-
-        if (hasErrors) {
-          console.error(
-            `React Router ERROR: ${JSON.stringify((reactRenderingResult as {routeError: Error}).routeError)}`,
-          );
-        }
+      // We let the client side handle any redirect
```

Comment:
whitespace changes caused by prettier

Reply:
Is this unavoidable?

Can we have a separate pr for prettier?

Reply:
reverted